backend/inspirehep/search/aggregations.py

```diff
@@ -31,7 +32,7 @@ def hep_doc_type_aggregation(order, title="Document Type", agg_type="checkbox"):
                 "bucket_help": {
                     "published": {
                         "text": "Published papers are believed to have undergone rigorous peer review.",
-                        "link": "https://inspirehep.net/info/faq/general#published",
+                        "link": f"{get_inspirehep_url()}/help/knowledge-base/faq/#faq-published",
```

Comment:
here I'm not sure if I should construct the url using the server name as this wouldn't work on QA since we have a qa faq published. wdyt?

Reply:
Actually leaving it ``https://inspirehep.net/help/help/knowledge-base/faq/#faq-published`` would work for both. Do we use it anywhere else? if yes, maybe we can add ```KNOWLEDGE_BASE_URL``` as the ui

Reply:
it's not used anywhere else. Yeah, I could change the link to remain with ```inspirehep.net``` but I still need to change to point to the new knowledge base. I was trying to avoid changing it so that in the future we wouldn't come fix hardcoded server names. But this won't work until we switch to the new domain.

Reply:
I updated it